python/graphframes/tests.py

```diff
@@ -227,6 +261,18 @@ def test_page_rank(self):
         pr = g.pageRank(resetProb, tol=errorTol)
         self._hasCols(pr, vcols=['id', 'pagerank'], ecols=['src', 'dst', 'weight'])
 
+    def test_parallel_personalized_page_rank(self):
+        if GraphFrameTestUtils.is_later_version("2.1"):
```

Comment:
actually, instead of a decorator, this can be called in code
https://docs.python.org/2/library/unittest.html#unittest.skipIf